server/boot/model-explorer.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function mountLoopBackExplorer(server) {
+  var explorer;
+  try {
+    explorer = require('loopback-explorer');
+    var restApiRoot = server.get('restApiRoot');
+
+    var explorerApp = explorer(server, {basePath: restApiRoot});
+    server.use('/explorer', explorerApp);
```

Comment:
- app() looks like it might be `loopback().rest()`, based on server/boot/rest-api.js